runtime/Cpp/runtime/src/Recognizer.h

```diff
@@ -11,7 +11,7 @@ namespace antlr4 {
 
   class ANTLR4CPP_PUBLIC Recognizer {
   public:
-    static const size_t EOF = std::numeric_limits<size_t>::max();
+    static const size_t EOF = (size_t)-1; // std::numeric_limits<size_t>::max(); doesn't work in VS 2013.
```

Comment:
Certainly doable, I just like to avoid lengthy cast expressions in simple cases like this (it's probably optimized away by the compiler).